utils/token_utils.go

```diff
@@ -13,19 +13,21 @@ import (
 
 // GetToken creates a JWT Token and returns it
 func GetToken(r *http.Request) (response *models.Token, err error) {
-	expiryTime := time.Now().Add(time.Hour * 24).Unix()
```

Comment:
This returns the expiryTime in seconds, whereas we are used to unix epoch. So converting.